conan/api/subapi/lockfile.py

```diff
@@ -27,7 +27,7 @@ def get_lockfile(self, lockfile=None, conanfile_path=None, cwd=None, partial=Fal
         :param conanfile_path: The full path to the conanfile, if existing
         :param lockfile: the name of the lockfile file
         """
-        if lockfile == "None":
+        if lockfile == "":
             # Allow a way with ``--lockfile=None`` to optout automatic usage of conan.lock
```

Comment:
Even apart of this comment:

```suggestion
            # Allow a way with ``--lockfile=""`` to optout automatic usage of conan.lock
```
Please, review the `get_lockfile()` docstring because it refers to the `None` value several times.

Another comment about it. I'm seeing the other API functions use as the first argument `lockfile`, and they distinguish the value `None` too, so I'm wondering if they should take into account this new empty value.